### PR TITLE
Add keys function

### PR DESCRIPTION
**Pull request by DiegoRBaquero**
Based on the discussion here: https://github.com/feross/bittorrent-tracker/pull/160/files/7f98203d5ec63f395e2794700a10046c54dcf8f1#diff-78c12f5adc1848d13b1c6f07055d996eR172

### CODE REVIEW

**Comment by chriso** [comment]
Can you update the README and add a test? Thanks.

File: index.js [hunk 1]
```diff
@@ -17,6 +17,10 @@ function LRU (opts) {
 
 inherits(LRU, events.EventEmitter)
 
+Object.defineProperty(LRU.prototype, 'keys', {
+  get: function () { return Object.keys(this.cache) }
+})
+
```
Comment (mafintosh) [inline index.js]: shouldn't this be a normal function instead?

Comment (DiegoRBaquero) [inline index.js]: I had it as a function (https://github.com/chriso/lru/pull/22/commits/01fca8c262475a5eec2bb9c553ccbe323af5540f) but thought it made more sense for it to be a property.

**Comment by DiegoRBaquero** [comment]
I have updated the tests to use the .keys property, removed one repeated test case and updated the README. The commits should be squashed

**Comment by chriso** [comment]
Thanks.

**Comment by mafintosh** [comment]
3.0.0

**Comment by feross** [comment]
👍

**Pull request merged**